comps/embeddings/multimodal/bridgetower/bridgetower_embedding.py

```diff
@@ -56,7 +56,8 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
         Returns:
             List of embeddings, one for each text.
         """
-        encodings = self.PROCESSOR.tokenizer(texts, return_tensors="pt").to(self.device)
+        encodings = self.PROCESSOR.tokenizer(
+            texts, return_tensors="pt", max_length=200, padding="max_length", truncation=True).to(self.device)
```

Comment:
This update was needed otherwise it would fail when each line of text wasn't the same exact length. I made the parameters the same as what's used in `embed_image_text_pairs` (see line 102-103 in this file)